app/helpers/connection_helper.rb

```diff
@@ -0,0 +1,61 @@
+module ConnectionHelper
+  def extract_conn_count(hash)
+    target_log = {}
+    hash.each{|instance, logs|
+      target_log[instance] = {}
+      logs.each{|log|
+        if log =~ /^I, \[([-\d]+)T([\d:]+)\.\d+\s#\d+\]\s+INFO -- : Connected from [\d.:]+\sI have (\d+) connections.$/
+          target_log[instance].store("#{$1} #{$2}", $3)
+        end
+      }
+    }
+    m_array = merge_connection_count(target_log)
```

Comment:
m_array means "merged_connection_count_array".
but I agree it is ambiguous.
so I changed to "conn_cnt_array".